src/commcare_cloud/ansible/roles/devops_scripts/tasks/main.yml

```diff
@@ -14,3 +15,60 @@
     owner: ansible
     group: ansible
     mode: "u=rwx,g=rwx,o=r"
+
+- name: Copy sudo-report.sh script to /usr/local/sbin
+  template:
```

Comment:
this could use the `copy` module since it's not actually a template

Reply:
used copy module.